packages/vuetify/src/components/transitions/createTransition.ts

```diff
@@ -45,59 +23,49 @@ export function createSimpleTransition (
       },
     },
 
-    render (h, context): VNode {
-      const tag = `transition${context.props.group ? '-group' : ''}`
-      const data: VNodeData = {
-        props: {
+    setup (props, { slots }) {
+      return () => {
+        const tag = props.group ? TransitionGroup : Transition
```

Comment:
can the user still supply a custom tag when using **group**?

Reply:
yes